api/base/filters.py

```diff
@@ -66,6 +69,33 @@ def filter_queryset(self, request, queryset, view):
             return queryset.sort(*ordering)
         return queryset
 
+    def check_serializer_fields(self, queryset, query_params, view, ordering):
```

Comment:
In general, where we're overriding DRF, I think we should try to stick as close to DRF as we can, rather than having the default be to roll our own solution. It's a well-tested framework, so I'd dig into their source code to see what we can use ourselves or what we might be able to override. Also, too many deviations are going to make this difficult to upgrade.